src/Container/NotFoundHandlerFactory.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
- * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
Why 2016 is removed? It could be updated to 2018.

Reply:
Why we do not have a script for the entire header? It is tiring to read, comment and fix always the same.

Reply:
Because we still don't have zend-coding-standard in place?
And as you can see different developers usually use different style here, different years, even they know how it should be.... Even in this PR we have different range year for new files, year changed here, not updated to the current year...

Reply:
I'm not sure why 2017 was used here; it should have been 2018; likely me still getting used to the new year.

This class is technically new, as it's now creating a `Zend\Expressive\Handler\NotFoundHandler` vs the old `Zend\Expressive\Middleware\NotFoundHandler`. The class below, `NotFoundMiddlewareFactory`, is essentially a rename of the latter factory.